acronym/acronym_test.rb

```diff
@@ -0,0 +1,14 @@
+class Acronym
+
+    def initialize(str)
+        @str = str
+    end
+
+    def abbreviate
+        @str.scan(/\w+/).join(" ").split.map(&:chr).join.upcase
```

Comment:
Just for comparison, my solution was
` str.gsub(/[^\w\s]/, ' ').split.map { |word| word.capitalize.chr }.join`

Reply:
Can I merge my solution to master now? Or you have to do it?